### PR TITLE
Wire up scan logic into CLI

### PR DESCRIPTION
**Pull request by holmboe**
We can now run `holmscan kaboom "Example Host"`

### CODE REVIEW

File: holmscan/cli.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 kaboom_args = """
 Usage:
-    holmscan kaboom [options]
+    holmscan kaboom [options] <asset>
 
 Options:
     -h, --help          Show this help message and exit
```
Comment (Grokzen) [inline holmscan/cli.py:48]: Don't you need to have `<asset>` here in the options menu? Describing what it is? or do docopt pick it up properly anyway?

File: holmscan/scan.py [hunk 2]
```diff
@@ -18,5 +18,71 @@ def __init__(self, controller):
...
+    @login
+    def start_web_scan(self, **kwargs):
+        data = kwargs
+        data["assets"] = kwargs["assets"]
+        data.update(
+            {
+                "appliance": "-1",
+                "source": 0,
+                "node": "",
+                "user": "{}".format(self.user_id),
```
Comment (Grokzen) [inline holmscan/scan.py]: `{0}`

File: holmscan/scan.py [hunk 3]
```diff
@@ -18,5 +18,71 @@ def __init__(self, controller):
...
+
+        resp = self.controller.session.post(
+            "https://sc.holmsecurity.com/scan/wscans/",
+            data=json.dumps(data).replace(" ", ""),
+            headers={"Content-Type": "application/json"},
+        )
+
+        response = json.loads(resp.text)
+        start_resp = self.controller.session.get(
+            "https://sc.holmsecurity.com/scan/start/{}/".format(response["id"])
```
Comment (Grokzen) [inline holmscan/scan.py]: `{0}`

File: holmscan/scan.py [hunk 4]
```diff
@@ -18,5 +18,71 @@ def __init__(self, controller):
...
+                "appliance": "-1",
+                "source": 0,
+                "node": "",
+                "user": "{}".format(self.user_id),
+                "start_date": None,
+            }
+        )
+
+        resp = self.controller.session.post(
+            "https://sc.holmsecurity.com/scan/wscans/",
```
Comment (Grokzen) [inline holmscan/scan.py]: Missing parameterized for `HOLMSEC_URL` ?

File: holmscan/scan.py (hunk 3 above)
Comment (Grokzen) [inline holmscan/scan.py]: Missing parameterized url `HOLMSEC_URL`

File: holmscan/scan.py [hunk 5]
```diff
@@ -18,5 +18,71 @@ def __init__(self, controller):
...
+        for webapp in webapps["results"]:
+            if name == webapp["name"]:
+                return webapp["asset"]["id"]
+
+        return None
+
+    @property
+    def user_id(self):
+        """Return the ID of the currently logged in user."""
+        url = u"{0}users/user/profile".format(self.controller.conf.get("HOLMSEC_URL"))
```
Comment (Grokzen) [inline holmscan/scan.py:83]: Idea is to move out common url formattinggs to method, maybe in controller class that formats `HOLMSEC_URL` by default or through using `{HOLMSEC_URL}` inside the input string.

**Review by Grokzen — COMMENTED** [review]
Some review comments

File: holmscan/cli.py (hunk 1 above)
Comment (holmboe) [reply holmscan/cli.py:48]: I looked at our other tools and we do not provide more info than that. Perhaps we should provide som example usage directly in the help output. Good point, but leaving for now.

File: holmscan/scan.py (hunk 5 above)
Comment (holmboe) [reply holmscan/scan.py:83]: Good idea, will implement in later PR.

**Review by Grokzen — APPROVED** [review]
LGTM

File: holmscan/scan.py (hunk 5 above)
Comment (holmboe) [reply holmscan/scan.py:83]: This is in #11

**Pull request merged**